src/main/groovy/com/getkeepsafe/dexcount/sdkresolver/SdkResolver.groovy

```diff
@@ -0,0 +1,122 @@
+package com.getkeepsafe.dexcount.sdkresolver
+
+import org.gradle.api.Project
+import org.gradle.api.logging.Logger
+import org.gradle.api.logging.Logging
+import org.gradle.api.tasks.StopExecutionException
+
+import static com.android.SdkConstants.ANDROID_HOME_ENV
+import static com.android.SdkConstants.FN_LOCAL_PROPERTIES
+import static com.android.SdkConstants.PLATFORM_WINDOWS
+import static com.android.SdkConstants.SDK_DIR_PROPERTY
+import static com.android.SdkConstants.currentPlatform
+
+/**
```

Comment:
The standard practice here is to keep the license at the very top of the file and, if anything has changed (which is the case here), adding a new line like so:

```
* Copyright 2016 KeepSafe Software
* Copyright 2014 Jake Wharton
* etc
```
Please do so with this and System.groovy.